<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_main</class>
 <widget class="QWidget" name="student_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="student_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>921</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="lexicon_frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="n_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>当前正在学习的词库</string>
       </property>
      </widget>
      <widget class="QLabel" name="lexicon_label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>131</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="change_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>更换词库</string>
       </property>
      </widget>
      <widget class="QPushButton" name="learn_btn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>290</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>开始学习</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>241</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>340</y>
         <width>121</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>更改背词计划</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>551</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="start_page">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>531</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string>尚未开始计划</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="word_page">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>531</width>
        <height>401</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="word_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>529</width>
         <height>399</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="info_page">
     <widget class="QFrame" name="detail_frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>531</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>511</width>
         <height>381</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>姓名：小明他王叔叔</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>一些图片</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>年龄：18</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>性别：沃尔玛购物袋</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>电话号码：110</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>备注：你无敌了孩子</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>对战积分：1000</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>学校：北京邮电大学包子铺</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="class_page">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>401</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="class_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>449</width>
         <height>399</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>暂未加入班级</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="info_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户id</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="min_button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>21</width>
      <height>8</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_5">
     <widget class="QFrame" name="btn_frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="battle_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>联机对战</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="setting_btn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>设置</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QFrame" name="btn_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="test_btn">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>自我测试</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="task_btn">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>任务</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="info_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>个人信息</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="class_btn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>班级</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="ud_frame">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="up_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="down_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>ud_frame</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>stackedWidget_2</zorder>
   <zorder>info_frame</zorder>
   <zorder>close_button</zorder>
   <zorder>min_button</zorder>
   <zorder>stackedWidget_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
